lib/aur-srcver

```diff
@@ -96,10 +96,10 @@ for n in "$@"; do
 done
 
 for f in "${failed[@]}"; do
-    warning 'makepkg %s failed for package %s\n' "${makepkg_args[*]}" "$f"
+    warning 'makepkg %s failed for package %s' "${makepkg_args[*]}" "$f"
 
     cat "$tmp/$f"/log >&2
-    printf '8<----\n' >&2
+    plain '8<----\n'  >&2
```

Comment:
Not sure if intentional, but `plain` already takes care of the trailing new line, this will add a empty line between messages.

Reply:
It wasn't, good catch.